actions/ocr_helpers.go

```diff
@@ -24,25 +24,59 @@ func DeployOCRContracts(
 	chainlinkNodes []client.Chainlink,
 	networks *client.Networks,
 ) []contracts.OffchainAggregator {
+	// Deploy contracts
 	var ocrInstances []contracts.OffchainAggregator
-	for i := 0; i < numberOfContracts; i++ {
+	for contractCount := 0; contractCount < numberOfContracts; contractCount++ {
 		ocrInstance, err := contractDeployer.DeployOffChainAggregator(
 			linkTokenContract.Address(),
 			contracts.DefaultOffChainAggregatorOptions(),
 		)
-		Expect(err).ShouldNot(HaveOccurred(), "Deploying OCR instance %d shouldn't fail", i+1)
+		Expect(err).ShouldNot(HaveOccurred(), "Deploying OCR instance %d shouldn't fail", contractCount+1)
+		ocrInstances = append(ocrInstances, ocrInstance)
+		if (contractCount+1)%100 == 0 { // For large amounts of contract deployments, space things out some
+			err = networks.Default.WaitForEvents()
+			Expect(err).ShouldNot(HaveOccurred(), "Failed to wait for OCR Contract deployments")
+		}
+	}
+	err := networks.Default.WaitForEvents()
+	Expect(err).ShouldNot(HaveOccurred(), "Error waiting for OCR contract deployments")
+
+	// Gather transmitter and address payees
+	var transmitters, payees []string
+	for _, node := range chainlinkNodes[1:] {
+		addr, err := node.PrimaryEthAddress()
+		Expect(err).ShouldNot(HaveOccurred(), "Error getting node's primary ETH address")
+		transmitters = append(transmitters, addr)
+		payees = append(payees, networks.Default.GetDefaultWallet().Address())
+	}
+
+	// Set Payees
+	for contractCount, ocrInstance := range ocrInstances {
+		err = ocrInstance.SetPayees(transmitters, payees)
+		Expect(err).ShouldNot(HaveOccurred(), "Error setting OCR payees")
+		if (contractCount+1)%100 == 0 { // For large amounts of contract deployments, space things out some
```

Comment:
Would you want to break this 100 out to a variable since it is consistent in 3 places and you likely would want to update all 3 to be the same if it needs to be updated in the future?